website/content/api-docs/index.mdx

```diff
@@ -47,7 +47,7 @@ There are five primary nouns in Nomad:
 - deployments
 - evaluations
 
-[![Nomad Data Model](/img/nomad-data-model.png)](/img/nomad-data-model.png)
+[![Nomad Data Model](https://nomadproject.io/img/nomad-data-model.png)](https://nomadproject.io/img/nomad-data-model.png)
```

Comment:
@ashleemboyer interesting, we don't want to rewrite this I would presume. Looks like the markdown syntax for the link is _off_ though.

Reply:
Syntax is right actually, just reads confusing since its an image link wrapped by a link

Reply:
I think we have a couple of things to do. Let's double check with @BRKalow on how images are referenced across content repos (is it consistent?), and then update the script to ignore internal links/paths to images. I'm pretty sure a marketing content workflow handles rewriting these in content before they reach the server-side plugins, so we shouldn't need to make any changes to the plugins.

Reply:
It depends on where authors put images in the `public` folder, but they are mostly consistent. We should be able to ignore any relative paths to images as part of the rewrites, like you said 👍

Reply:
Update: we decided to fix the links in this repo manually, then we'll circle back to update the script.